apps/test/unit/gamelab/GameLabSpriteTest.js

```diff
@@ -1358,21 +1358,28 @@ describe('GameLabSprite', function () {
       sprite.setCollider('circle');
       expect(sprite.collider).to.be.an.instanceOf(gameLabP5.p5.CircleCollider);
       expect(sprite.collider.center).to.eq(sprite.position);
+      // Radius should be half of sprite's larger dimension.
+      expect(sprite.collider.radius).to.eq(sprite.height / 2);
+      // Offset should be zero
       expect(sprite.collider.offset).to.be.an.instanceOf(p5.Vector);
       expect(sprite.collider.offset.x).to.eq(0);
       expect(sprite.collider.offset.y).to.eq(0);
-      // Radius should be half of sprite's larger dimension.
-      expect(sprite.collider.radius).to.eq(20);
+    });
+
+    it('scaling sprite without animation does not affect default circle collider size', function () {
```

Comment:
This changes the behaviour right? In this example, circle colliders scale with the sprite: https://studio.code.org/projects/gamelab/_lp_WNTvmL5F27cHcedweg/edit